src/TornadoD3.ts

```diff
@@ -0,0 +1,402 @@
+        .append<SVGElement>('g')
+        .attr('class', 'histogram-container');
+
+      this.update(el, newProps);
+    },
+
+    /**
+    * Draw Axes
+    */
+    drawAxes() {
+      const { bar, data, groupLayout, margin, width, height, axis } = props;
+      const SPLIT_AXIS_HEIGHT = 20;
+      const valuesCount = maxValueCount(data.counts);
+      const w = gridWidth(props);
+      const h = gridHeight(props) - SPLIT_AXIS_HEIGHT;
+      const dataLabels = data.counts.map((c) => c.label);
+
+      y.domain(data.bins)
+        .rangeRound([h, 0])
+        .paddingInner(groupedMargin(bar));
+
+      innerScaleBand
+        .domain(groupedBarsUseSameXAxisValue({ groupLayout }) ? ['main'] : dataLabels)
+        .rangeRound([0, y.bandwidth()])
+        .paddingInner(barMargin(props.bar));
+
+      const xAxis = axisBottom<number>(x)
+        .tickFormat((v) => v.toString().replace('-', ''));
+      const yAxis = axisLeft<string>(y);
+      this.calculateDomain();
+
+      const x2 = scalePoint<any>();
+
+      const xGroupAxis = axisBottom(x2).tickPadding(SPLIT_AXIS_HEIGHT)
+        .tickSize(0)
+
+      x2.range([Number(width) / 4, Number(width) * (3 / 4) - (margin.top * 2) - axis.y.width])
+        .domain(props.splitBins);
+
+      /** Y-Axis (label axis) set up */
+
+      ticks({
+        axis: yAxis,
+        valuesCount,
+        axisLength: h,
+        axisConfig: axis.y,
+        scaleBand: y,
+        limitByValues: true,
+      });
+
+      yAxisContainer
+        .attr('transform', 'translate(' + yAxisWidth(axis) + ', ' + margin.top + ' )')
+        .call(yAxis);
+
+      // @TODO - Stacked? (was using appendDomainRange())
+      x.range([0, Number(width) - (margin.top * 2) - axis.y.width])
+        .domain(domain)
+        .nice();
+
+      const xAxisY = height - xAxisHeight(props.axis) - margin.top - SPLIT_AXIS_HEIGHT;
+      xAxisContainer
+        .attr('transform', 'translate(' + yAxisWidth(axis) + ',' +
+          xAxisY + ')')
+        .call(xAxis);
+
+      xAxisContainer2
+        .attr('transform', 'translate(' + yAxisWidth(axis) + ',' +
+          (xAxisY) + ')')
+        .call(xGroupAxis);
+
+      attrs(svg.selectAll('.y-axis .domain, .y-axis .tick line'), axis.y.style);
+      attrs(svg.selectAll('.y-axis .tick text'), axis.y.text.style as any);
+
+      attrs(svg.selectAll('.x-axis .domain, .x-axis .tick line'), axis.x.style);
+      attrs(svg.selectAll('.x-axis .tick text'), axis.x.text.style as any);
+    },
+
+    calculateDomain() {
+      const { data, center } = props;
+      const leftValues = data.counts.reduce((prev, next) => prev.concat(next.data[0]), [] as number[]);
+      const rightValues = data.counts.reduce((prev, next) => prev.concat(next.data[1]), [] as number[]);
+      domain = [-Math.max(...leftValues), Math.max(...rightValues)];
+      // Center the 0 axis value in the middle of the chart
+      if (center) {
+        const max = Math.max(Math.max(...leftValues), domain[1]);
```

Comment:
Small nitpick, this could just be `Math.max(domain[1], ...leftValues)`

Reply:
nope it can't little minus sign on line 233 means it wouldn't be the same value